test/example/current-user-async-pipe/current-user-async-pipe.component.ts

```diff
@@ -10,7 +10,7 @@ import { profile } from '../stores/profile';
 })
 export class CurrentUserAsyncPipeComponent {
   currentUser$: Observable<IUser> = this.nanostores.useStore(profile)
-    .pipe(switchMap(userId => this.nanostores.useStore(User(userId))));
+    .pipe(switchMap(({ userId }) => this.nanostores.useStore(User(userId))));
```

Comment:
That's great, thanks for the attentiveness 👍

Reply:
Thanks, but i actually had to because of the generic type =)